compiler/p/codegen/OMRTreeEvaluator.cpp

```diff
@@ -6220,3 +6220,41 @@ TR::Register *OMR::Power::TreeEvaluator::retrieveTOCRegister(TR::Node *node, TR:
 {
    return cg->machine()->getPPCRealRegister(TR::RealRegister::gr2);
 }
+
+TR::Register * OMR::Power::TreeEvaluator::ibyteswapEvaluator(TR::Node *node, TR::CodeGenerator *cg)
+{
+   TR_ASSERT(node->getNumChildren() == 1, "Wrong number of children in inlineIntegerReverseBytes");
+
+   TR::Node *firstChild = node->getFirstChild();
+   TR::Register *tgtRegister = cg->allocateRegister();
+
+   if (TR::Compiler->target.cpu.id() >= TR_PPCp7 &&
```

Comment:
There is no reason for (>=p7) to be there for lwbrx/stwbrx/lhbrx/sthbrx instructions.  But it should be there for ldbrx/stdbrx instructions.  Please fix it consistently everywhere ...